strawberry/schema/schema_converter.py

```diff
@@ -59,6 +59,10 @@
 from .types.concrete_type import ConcreteType
 
 
+# Extension key used to link a GraphQLType back into the Strawberry definition
+STRAWBERRY_DEFINITION = "strawberry-definition"
```

Comment:
Can we give this a more descriptive name?

Also can we put this on some namespace other than the module? Do you think it would make sense to be on `GraphQLCoreConverter`?

What do you think of `GraphQLCoreConverter.DEFINITION_BACKREF`?